### PR TITLE
fix(resilience): widen low-carbon content-age budget 36mo→60mo (interim, #4219)

### PR DESCRIPTION
**Pull request by koala73**
## Summary

`/api/health.lowCarbonGeneration` reads **`STALE_CONTENT`** — content age ~**53 months** vs the 36-month budget.

**Genuine World Bank upstream lag, not a seeder bug.** `resilience:low-carbon-generation:v1` sums three WB electricity indicators (`EG.ELC.NUCL.ZS` + `RNEW.ZS` + `HYRO.ZS`) at their **latest common year**, which is **2021** (one component — hydro/nuclear — is frozen there in WB WDI). The seeder correctly picks the latest common year; 2021 *is* the newest WB publishes. The original 36mo budget assumed an ~18mo per-component lag that doesn't hold for the min-common-year composite when one series stalls.

## Change (interim stopgap)

- `scripts/seed-low-carbon-generation.mjs`: `MAX_CONTENT_AGE_MIN` 36mo → **60mo**. Clears the warn while still tripping a real regression (a 2020 freeze is ~64mo > 60mo).
- This **inverts** the low-carbon vs fossil-share budget ordering (low-carbon 60mo > fossil 48mo) because low-carbon genuinely lags *more* than fossil (FOSL ~2023). Tests updated to match.

## Tests

- `tests/wb-country-dict-content-age.test.mjs`: budget → 60mo; refreshed the boundary cases; **added a frozen-at-2021 (~53mo) regression guard** documenting why the bump exists; rewrote the per-seeder separation test for the inverted ordering (52mo trips fossil-48 but not low-carbon-60).
- Verified: content-age 15 ✓, seeder 3 ✓, resilience doc-parity/registry/energy-v2/power-reliability 84 ✓, seeder-content-age-coverage 4 ✓, typecheck ✓, lint ✓. (Pre-push gate passed.)

## Durable follow-up

This is a **stopgap** — it widens tolerance, it doesn't refresh the data. The real fix (migrate low-carbon generation to Ember/OWID, which publish 2023+ electricity mix, and restore a tight ~13–18mo budget) is tracked in **#4219**.

### CODE REVIEW

File: tests/wb-country-dict-content-age.test.mjs [hunk 1]
```diff
@@ -114,30 +117,33 @@ test('low-carbon: fresh-arrival regression guard — max year 2024 (~17mo) withi
...
   const ageMin = (FIXED_NOW - cm.newestItemAt) / 60000;
   assert.ok(
     ageMin < LOW_CARBON_BUDGET,
-    `${Math.round(ageMin / 60 / 24 / 30)}mo < 36mo budget — within steady-state ceiling`,
+    `${Math.round(ageMin / 60 / 24 / 30)}mo < 60mo budget — the frozen-2021 live state must not false-positive`,
   );
 });
 
-test('low-carbon: catastrophic stall — max year 2020 (~65mo) trips STALE_CONTENT', () => {
+test('low-carbon: catastrophic stall — max year 2020 (~64mo) trips STALE_CONTENT', () => {
```
[the remark this answers is not in the document]
Comment (koala73) [reply tests/wb-country-dict-content-age.test.mjs]: Good catch — fixed in the latest push. Reverted the title + comment to ~65mo to match the computed value (Math.round(1951.5/30)=65) and the dynamic error string.
